tests/_support/Step/Acceptance/Installation/Installation.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace Step\Acceptance\Installation;
+
+use Page\Acceptance\Administrator\LoginPage;
+use Page\Acceptance\Administrator\ControlPanelPage;
+
+class Installation extends \AcceptanceTester
+{
+	/**
+	 * @Given Joomla CMS is installed
+	 */
+	public function joomlaCMSIsInstalled()
+	{
+		$I = $this;
+		// @todo: the following path should be taken from configuration and not hardcoded as it is now.
+		$I->assertFileExists('tests/joomla-cms3/configuration.php');
```

Comment:
We can have alternative ways of checking if Joomla is installed, I don't generally run the test on Joomla which is present in tests/joomla-cms3/ , like you mentioned in todo. I would prefer to move this assertion to something else and independent of the path.